server/app.js

```diff
@@ -22,46 +22,46 @@ import profileRouter from "./routes/profile";
 
 var app = express();
 
-const storage = multer.diskStorage({
-  destination: (req, file, cb) => cb(null, "images"),
-  filename: (req, file, cb) => cb(null, uuidv4())
-});
+// const storage = multer.diskStorage({
```

Comment:
Try not to leave in any commented code for PR, git history will help us get back thrown away code if needed